bin/css-validator

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+`echo eval exec node $(printf %q\  "$0" "$@")`
+const fs = require('fs')
+const validateCss = require('css-validator')
+
+validateCss({text: fs.readFileSync(process.argv[2])}, (err, data) => {
+  if (err) console.error(err)
```

Comment:
I think if there's an error, then we should throw it. It will provide an exit code still and give a much broader stack trace